### PR TITLE
Display logged user

### PR DESCRIPTION
**Pull request by ivanzusko**

### CODE REVIEW

File: src/helpers/parseJwt.js [hunk 1]
```diff
@@ -0,0 +1,6 @@
+export const parseJwt = token => {
+    const base64Url = token.split('.')[1];
```
Comment (italolelis) [inline src/helpers/parseJwt.js]: Please use a library to parse JWT. Maybe this one https://github.com/auth0/node-jsonwebtoken

Comment (ivanzusko) [reply src/helpers/parseJwt.js]: Done, @italolelis

File: src/modules/Layout/Header/Header.js [hunk 2]
```diff
@@ -7,7 +8,12 @@ import './Header.css';
 
 const b = block('j-header');
 
-const Header = ({ logged }) => {
+const propTypes = {
+    logged: PropTypes.bool.isRequired,
```
Comment (Vxplus) [inline src/modules/Layout/Header/Header.js]: if you already have the user wouldn't that already imply that it is logged in?

File: src/modules/Layout/Header/Header.js [hunk 3]
```diff
@@ -18,6 +24,12 @@ const Header = ({ logged }) => {
                     <Nav />
                 </div>
                 <div className={b('col', { right: true })}>
+                    {
+                        user &&
+                            <span className={b('user')}>
```
Comment (Vxplus) [inline src/modules/Layout/Header/Header.js]: maybe `user-name` or `display-name` would be more specific for this user sounds like a container imo.

File: src/modules/Layout/Header/Header.js [hunk 4]
```diff
@@ -7,7 +8,12 @@ import './Header.css';
 
 const b = block('j-header');
 
-const Header = ({ logged }) => {
+const propTypes = {
+    logged: PropTypes.bool.isRequired,
+    user: PropTypes.string.isRequired,
```
Comment (Vxplus) [inline src/modules/Layout/Header/Header.js]: maybe `username` fits this more.

File: src/modules/Layout/Header/Header.js (hunk 2 above)
Comment (ivanzusko) [reply src/modules/Layout/Header/Header.js]: actually yes, @Vxplus , totally agree!

File: src/modules/Layout/Header/Header.js (hunk 4 above)
Comment (ivanzusko) [reply src/modules/Layout/Header/Header.js]: maybe in future, there will appear some additional info, so I will change `user<String>` to `user<Object>` and fill it with all necessary info. What do you think, @Vxplus ?

Comment (Vxplus) [reply src/modules/Layout/Header/Header.js]: Fine by me.

**Pull request merged**